tools/FhirSchemaManager/Model/ErrorDescription.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net;
+using EnsureThat;
+
+namespace FhirSchemaManager.Model
+{
+    public class ErrorDescription
+    {
+        public ErrorDescription(int statusCode, string message)
```

Comment:
You could make all of your exceptions have a base type of something like `SchemaManagerException` and pass that into `RenderErrror` instead of having this intermediate class.

Also, you may just be able to get away with passing all exceptions to `RenderError` and special case the schema manager specific ones.